### PR TITLE
README.md: Point contributors to Linux vs. Mac contribution guidelines

### PR DESCRIPTION
**Pull request by bw-yuhogg**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/brew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?

---

Update README to point users to the Linux vs. Mac contribution guidelines. We don't want upstream to miss important pull requests nor to receive undesired Linux-only PR's.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ We'd love you to contribute to Linuxbrew or its upstream, Homebrew. First, pleas
 
 We explicitly welcome contributions from people who have never contributed to open-source before: we were all beginners once! We can help build on a partially working pull request with the aim of getting it merged. We are also actively seeking to diversify our contributors and especially welcome contributions from women from all backgrounds and people of colour.
 
-Please note that contributions of new formulae or version updates should usually go upstream to Homebrew first.
+Please also see our [guidelines](https://github.com/Linuxbrew/brew/blob/master/.github/CONTRIBUTING.md#contributing-to-linuxbrew) on whether to send pull requests to Linuxbrew or Homebrew.
```
Comment (sjackman) [inline README.md]: Remove the word `also`. Looks good to me otherwise. @mikemcquaid ?

Comment (MikeMcQuaid) [inline README.md]: 👍 thanks!

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ We'd love you to contribute to Linuxbrew or its upstream, Homebrew. First, pleas
```
Comment (bw-yuhogg) [inline README.md:113]: Just noticed that we should probably update the "Something broke!" section to reflect that there are now 2 repos (core vs brew)

**Comment by sjackman** [comment]
Thanks, Bob!

**Pull request merged**